rl_studio/agents/robot_mesh/manual_pilot.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+import time
+
+import sys as system
+from envs.gazebo_envs import *
+
+import numpy as np
+from functools import reduce
+
+from agents.f1.settings import QLearnConfig
+
+# from gym.envs.registration import register
+
+# # my envs
+# register(
+#     id='mySim-v0',
+#     entry_point='envs:MyEnv',
+#     # More arguments here
+# )
+
```

Comment:
Does this code not belong to the registry file?

Reply:
Yes. I changed and test it today